etl/definitions.py

```diff
@@ -23,7 +23,7 @@
             data_file_names_default=("mini-wikipedia.output.txt",),
             distance_strategy_default=DistanceStrategy.EUCLIDEAN_DISTANCE,
```

Comment:
Some of these variables aren't obviously "input". Make one or more additional configs?